proto/store/setting.proto

```diff
@@ -261,10 +261,20 @@ message MaskingAlgorithmSetting {
       string salt = 1;
     }
 
+    message InnerOuterMask {
+      int32 prefix_len = 1;
+      int32 suffix_len = 2;
```

Comment:
And if we won't have the `prefix_len` and `suffix_len` at the same time, we only need the `type` and `mask_length`

Reply:
I see. Forget this comment